templates/shortcodes/embed_video.html

```diff
@@ -0,0 +1,12 @@
+<!-- {#
```

Comment:
Double comments explained in ef98066a6c3bcd30eb8af568d8b092476571ef13

Reply:
Makes sense. Thanks for adding the shortcode :)